components/WeeklyCard/index.js

```diff
@@ -1,19 +1,31 @@
+import Link from "next/link";
 import styled from "styled-components";
-const StyledListItem = styled.li`
-  font-size: 0.6rem;
+const StyledLink = styled(Link)`
+  color: inherit;
+  text-decoration: none;
   display: grid;
   grid-template-columns: 1fr 3fr 1fr 1fr;
 `;
+const StyledListItem = styled.li`
+  font-size: 0.6rem;
+  border-radius: 4px;
+  transition: background-color 0.2s ease;
+  &:hover {
+    background-color: lightslategray;
+  }
+`;
 export default function WeeklyCard({ event }) {
   return (
     <StyledListItem>
-      <p>{event.time}</p>
-      <p>{event.title}</p>
-      <p>
-        {event.country}
-        {event.flag}
-      </p>
-      <p>{event.impact}</p>
+      <StyledLink href={`/events/${event.id}`}>
```

Comment:
See above ^^

Also, since the content of the `StyledLink` are shared between the `WeeklyCard` and the `TodaysCard`, you can abstract them in a separate component that you can reuse, if you don't plan any different behaviours.